src/org/zaproxy/zap/extension/ascanrulesAlpha/IntegerOverflow.java

```diff
@@ -0,0 +1,271 @@
+    		if (msg.getResponseHeader().getStatusCode() == HttpStatusCode.INTERNAL_SERVER_ERROR )
+    		{
+    			log.debug("Found Header");
+    			bingo(getRisk(), 
+    					Alert.CONFIDENCE_MEDIUM, 
+    					this.getBaseMsg().getRequestHeader().getURI().toString(), 
+    					param, 
+    					msg.getRequestHeader().toString(), 
+    					this.getError('1') ,
+    					msg);
+    			return;
+    		}
+    		msg = getNewMsg();
+			returnAttack = singleString(4,'0');// The number of full length ints to send.
+			setParameter(msg, param, returnAttack);
+			sendAndReceive(msg);
+			requestReturn = msg.getResponseHeader();			
+			// This is where BASE baseResponseBody was you detect potential vulnerabilities in the response
+    		chkerrorheader = requestReturn.getHeadersAsString();
+    		log.debug("Header: "+ chkerrorheader);
+    		if (msg.getResponseHeader().getStatusCode() == HttpStatusCode.INTERNAL_SERVER_ERROR)
+    		{
+    			log.debug("Found Header");
+    			bingo(getRisk(), 
+    					Alert.CONFIDENCE_MEDIUM, 
+    					this.getBaseMsg().getRequestHeader().getURI().toString(), 
+    					param, 
+    					msg.getRequestHeader().toString(), 
+    					this.getError('2') ,
+    					msg);
+    			return;
+    		}
+    		msg = getNewMsg();
+			returnAttack = singleString(4,'1');// The number of full length ints to send.
+			setParameter(msg, param, returnAttack);
+			sendAndReceive(msg);
+			requestReturn = msg.getResponseHeader();			
+			// This is where BASE baseResponseBody was you detect potential vulnerabilities in the response
+    		chkerrorheader = requestReturn.getHeadersAsString();
+    		log.debug("Header: "+ chkerrorheader);
+    		if (msg.getResponseHeader().getStatusCode() == HttpStatusCode.INTERNAL_SERVER_ERROR)
+    		{
+    			log.debug("Found Header");
+    			bingo(getRisk(), 
+    					Alert.CONFIDENCE_MEDIUM, 
+    					this.getBaseMsg().getRequestHeader().getURI().toString(), 
+    					param, 
+    					msg.getRequestHeader().toString(), 
+    					this.getError('3') ,
+    					msg);
+    			return;
+    		}
+    		msg = getNewMsg();
+			returnAttack = singleString(4,'9');// The number of full length ints to send.
+			setParameter(msg, param, returnAttack);
+			sendAndReceive(msg);
+			requestReturn = msg.getResponseHeader();			
+			// This is where BASE baseResponseBody was you detect potential vulnerabilities in the response
+    		chkerrorheader = requestReturn.getHeadersAsString();
+    		log.debug("Header: "+ chkerrorheader);
+    		if (msg.getResponseHeader().getStatusCode() == HttpStatusCode.INTERNAL_SERVER_ERROR)
+    		{
+    			log.debug("Found Header");
+    			bingo(getRisk(), 
+    					Alert.CONFIDENCE_MEDIUM, 
+    					this.getBaseMsg().getRequestHeader().getURI().toString(), 
+    					param, 
+    					msg.getRequestHeader().toString(), 
+    					this.getError('4') ,
+    					msg);
+    			return;
+    		}
+    			
+				return;	
+
+			
+		} catch (Exception e) {

```

Comment:
Actually I checked and this is required by the sendAndReceive(msg).  Some kind of try/catch is required because of the exception thrown so I'm really not sure if it can be removed as this is what eclipse suggests is the proper way to do it.